aframe/tasks/train/train.py

```diff
@@ -109,13 +109,13 @@ def get_config(self):
         return json_string
 
     def get_args(self):
-        # get args from base class removing the first two
-        # which reference the config file. We'll
-        # need to set this as an environment variable of
-        # raw yaml content to run remotely.
+        # get args from local train base class, removing the first three
+        # which reference the path to a local config file.
+        # For remote training, we'll need to set the config
+        # as an environment variable of raw yaml content
         args = super().get_args()
         args = args[2:]
```

Comment:
I think this needs to be `[3:]`

Reply:
Oh wait, nvm

Reply:
yeah - I removed the `fit` subcommand from the args, I thought it was clearer that way